x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/dev_only.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import React, { memo, useEffect } from 'react';
+import type { CommandArgumentValueSelectorProps } from '../../console/types';
+import type { CommandDefinition } from '../../console';
+import { ArgumentFileSelector } from '../../console_argument_selectors';
+
+// FOR DEV PURPOSES ONLY. WILL BE DELETED PRIOR TO MERGE
+// FIXME:PT DELETE FILE
+export const getUploadCommand = ({
```

Comment:
This needs to be cleaned up before merging. No?

Reply:
Yes, but I want to do it in a subsequent PR to follow closely behind this one. I have some other changes that I want to see if I can get in and will need this Dev command to do test it out.